mkdocs.yml

```diff
@@ -696,6 +697,7 @@ plugins:
       intelligence_center/integrations/opencti.md: cti/features/integrations/opencti.md
       intelligence_center/integrations/splunk.md: cti/features/integrations/splunk.md
       intelligence_center/integrations/thehive.md: cti/features/integrations/thehive.md
+      intelligence_center/integrations/threatquotient.md: cti/features/integrations/threatquotient.md
```

Comment:
This is not necessary to add a redirect since the page is new (no need to redirect dangling links to the new page).

```suggestion
```